### PR TITLE
updates for grammar and flow.

### PR DESCRIPTION
**Pull request by yulgroper**

### CODE REVIEW

**Comment by smokhov** [comment]
@yulgroper -- there are a couple of issues here.

- please make sure you normally pull first then do the updates or merge the upstream changes into your branch first before editing and submitting a pull
- please do not commit .pdf at the same time as .tex (the proposed .pdf conflicts as a result and has outdated info compared, to what's in master, e.g., the version number). We usually do PDFs last thing when a bulk is done and looks more polished, like a revision, which is 6.6-dev-03, 04, etc. and not 6.7 at this time :)

File: doc/speed-manual.tex [hunk 1]
```diff
@@ -219,68 +219,111 @@ \section{Job Management}
...
-are broken into four main sections: make a copy of \file{template.sh}
-or any existing job script example on our GitHub, and base
-your job on that. The first section is the shell call, and that can be
-left alone, unless you want to use a different shell. The next section
-contains the options provided to the cluster scheduler, and \file{template.sh}
-provides the essentials:
+Preparing your job for submission is fairly straightforward. Editing a copy
+of the\file{template.sh} you moved into your home directory during section
+\refname{envsetup} is a good place to start. You can also use a job script
+example from our GitHub's (\url{https://github.com/NAG-DevOps/speed-hpc}) ''src''
```
Comment (smokhov) [inline doc/speed-manual.tex:303]: Proper opening double quote in LaTeX is `` (two backticks), not ''

File: doc/speed-manual.tex [hunk 2]
```diff
@@ -363,9 +409,9 @@ \subsection{Job Submission Basics}
 provided to the scheduler option, \option{-pe smp}), though, obviously, every
 program has its own unique execution framework.
 
-Please note that jobs that expect to touch data-input and -output files
+Please note that jobs that expect to touch data-input and data-output files
 more than once should make use of \api{TMPDIR}, a scheduler-provided working
-space (this is quite generous, close to 1~TB in size). \api{TMPDIR} is created
+space (this is quite generous, close to ~1TB in size). \api{TMPDIR} is created
```
Comment (smokhov) [inline doc/speed-manual.tex:414]: `1~TB` is actually correct, here is the ~ is a non-breaking space and not an "approximate" sign.

**Review by smokhov — CHANGES_REQUESTED** [review]

**Comment by smokhov** [comment]
You'd also have to edit your commit to remove the .pdf from it and to force-push it to resolve the conflicts.

File: doc/speed-manual.tex [hunk 3]
```diff
@@ -219,68 +219,111 @@ \section{Job Management}
...
-Preparing your job for submission is fairly straightforward. Job scripts
-are broken into four main sections: make a copy of \file{template.sh}
-or any existing job script example on our GitHub, and base
-your job on that. The first section is the shell call, and that can be
-left alone, unless you want to use a different shell. The next section
-contains the options provided to the cluster scheduler, and \file{template.sh}
-provides the essentials:
+Preparing your job for submission is fairly straightforward. Editing a copy
+of the\file{template.sh} you moved into your home directory during section
+\refname{envsetup} is a good place to start. You can also use a job script
```
Comment (smokhov) [inline doc/speed-manual.tex:302]: Missing space before \file and instead of \refname please use \xs

Comment (yulgroper) [reply doc/speed-manual.tex:302]: Nice catch. Will use \xs going forward

**Comment by yulgroper** [comment]
I had not intended to try to commit the pdf and am not sure how to remove it from this pull request :(.

**Comment by yulgroper** [comment]
I've added *.pdfs to my local .gitignore file. However, I think it is better to close this pull request and try again.

**Pull request closed without merging**